exec/java-exec/src/test/java/org/apache/drill/exec/ExecTest.java

```diff
@@ -120,4 +124,27 @@ protected LogicalExpression parseExpr(String expr) throws RecognitionException {
     return ret.e;
   }
 
+  /**
+   * mock current locale to US
```

Comment:
Please start the sentence with a capital letter and add a description about the method which is mocked.